shell/src/main/java/alluxio/cli/LogLevel.java

```diff
@@ -180,7 +180,8 @@ private static String parseOptLevel(CommandLine cmd) {
     return null;
   }
 
-  private static void setLogLevel(final TargetInfo targetInfo, String logName, String level)
+  private static void setLogLevel(final TargetInfo targetInfo, String logName, String level,
+                                  AlluxioConfiguration alluxioConf)
```

Comment:
This passes the conf to the command internal HTTP call, in case a parameter should be sent to the server.